src/Translator.js

```diff
@@ -544,6 +554,46 @@ class Translator extends EventEmitter {
     return this.resourceStore.getResource(code, ns, key, options);
   }
 
+  getUsedParamsDetails(options = {}) {
+    const optionsKeys = [
```

Comment:
we need to remember to extend this array if in future there will be more option properties...

Reply:
yeah, I was thinking about pulling those keys from somewhere to not have to remember to update it. Is there a place I could pull them from?

Reply:
Can you at least create a comment there?

Reply:
added